includes/block-patterns.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Register and enqueue all things block patterns.
+ *
+ * @package tenup_podcasting
+ */
+
+namespace tenup_podcasting\block_patterns;
+
+/**
+ * Register block and its assets.
+ */
+function init() {
+	register_block_pattern(
+		'podcasting/podcast-grid',
+		array(
+			'title'       => __( 'Podcast Grid', 'simple-podcasting' ),
+			'description' => _x( 'Podcast Grid', 'This block pattern is used to display podcast in a grid structure.', 'simple-podcasting' ),
+			'content'     => '<!-- wp:query {"queryId":14,"query":{"perPage":3,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":false},"displayLayout":{"type":"flex","columns":3}} -->
```

Comment:
The `queryId` property probably needs to be removed here, it will be auto-assigned in the block editor once added.

Reply:
I made some changes:
- Use podcast terms for query
- Remove query ID
- Adds podcast name
- Use cover block featured image
- Change category for the pattern to query (included in core)

Without building a custom block for it, listening time is not possible with existing core blocks.